mara_app/.scripts/init.mk

```diff
@@ -7,8 +7,9 @@ SHELL=.scripts/mara-app/makeshell $(or $@,-)
 .SILENT:
 
 # copy scripts from mara-app package to project code
+.copy-mara-app-scripts: MODULE_LOCATION != .venv/bin/python -m pip show mara-app | sed -n -e 's/Location: //p'
 .copy-mara-app-scripts:
```

Comment:
Is this intended that there are now two lines?

Reply:
Yes. This defines a local variable for the Makefile target .copy-mara-app-scripts. This is done that we don’t get into issues with other mara packages where maybe the same variable could be used.